nh_clinical/tests/common/nh_clinical_test_utils.py

```diff
@@ -166,3 +198,32 @@ def copy_instance_variable_if_exists(self, caller, variable_name):
         instance_variable_value = getattr(self, variable_name, None)
         if instance_variable_value:
             setattr(caller, variable_name, instance_variable_value)
+
+    # TODO EOBS-753: Move to patient model? Is just a generally useful method.
```

Comment:
TBH this method's kind of misnamed as it's not looking at the `patient_id` but the `parent_id` (which is the spell's activity ID).

Additionally there's a few defaults in there such as `self.nurse.id` that make it a little hard to move into the patient model